view/components/filter-bar/select-date-range.js

```diff
@@ -118,6 +119,20 @@ module.exports = function (/* opts */) {
 			$.dispatchEvent($('startId'), 'change');
 			$.dispatchEvent($('endId'), 'change');
 		});
+
+		var path;
+		path = location.pathname;
+		location.on('change', function () {
+			if (location.pathname !== path) {
+				path = location.pathname;
```

Comment:
if path from the top is always location.pathname then 
location.pathname !== path ==> location.pathname !== location.pathname does not make any sence ie it is always false